types/node/child_process.d.ts

```diff
@@ -78,6 +78,20 @@ declare module "child_process" {
         prependOnceListener(event: "message", listener: (message: any, sendHandle: net.Socket | net.Server) => void): this;
     }
 
+    // return this object when stdio option is undefined or not specified
+    interface ChildProcessWithoutNullStreams {
```

Comment:
Properties like `killed`, `pid` are missing here. I think some common part should be used or this type only specializes the stdio relevant part of `ChildProcess`.

Reply:
It was meant to `extends ChildProcess` but I forgot.

Reply:
Fixed with https://github.com/DefinitelyTyped/DefinitelyTyped/pull/34285/commits/0237165eb143dbddfcc45a403df34cf8cc010a33